### PR TITLE
Fix mailto in contact page

### LINKED ISSUE #435: No mailto in the 'href' attribute on the contact page
Clicking on any mail-id on the contact page gives error this is because the 'href' attribute does not have "mailto:" attached to it. I would like to work on this issue.
@smit1678 @ramyaragupathy @dakotabenjamin Please approve this as an issue.

### PR DESCRIPTION
**Pull request by bismitaguha**
Fixes #435 . No error comes now when email link is clicked.
@ramyaragupathy Please reveiw this PR and provide suggestions.

### CODE REVIEW

**Comment by bismitaguha** [comment]
@ramyaragupathy this has fixed the mail id problems in all the pages.

**Review by ramyaragupathy — COMMENTED** [review]
Media section's mail id still needs to be worked upon @codefab99 

image

File: _sass/base.scss [hunk 1]
```diff
@@ -4647,7 +4647,8 @@ sup {
     }
   }
   .block-screenshot {
-    padding: 80px 0 0 80px;
+    padding-left: 40px;
```
Comment (ramyaragupathy) [inline _sass/base.scss]: Wondering what are these changes for?

File: _sass/base.scss [hunk 2]
```diff
@@ -4767,6 +4768,27 @@ sup {
 
 
 .tools {
+  .half-block .block-details {
+    display: flex;
+    flex-direction: column;
+    width: 825.455px;
+    @media (min-width: $screen-sm) {
+      padding: 60px 60px 40px 40px;
+      h3 {
```
Comment (ramyaragupathy) [inline _sass/base.scss]: I believe these are not needed the mail id update

**Comment by bismitaguha** [comment]
I have fixed the problem. Please review the PR @ramyaragupathy . I am working on the media page mail-id.

**Review by ramyaragupathy — APPROVED** [review]

**Pull request merged**